### ISSUE TITLE
поддержка centos, oel, redhat support

### ISSUE THREAD

**Issue by grebnebo** [bitrixdock/php71-fpm#1]
Привет! Просмотрели ваши образы - понравилась идея. Но есть одно но.
Вы забираете базовый образ у не пойми кого - phpdockerio/php71-fpm
Он это соответственно собирает только на ubuntu, возник вопрос про rpm-based дистрибутивы.
Например для продакшна практически не используется ubuntu, для установки доп. модулей - другие команды, вопрос унификации образов docker. Планируется ли поддержка rpm дистрибутивов? И включение xdebug в базовом образе - снижает производительность cpu раза в 3, это образы только для разработчиков получается, а как вы тогда их в продакшне используете? Или xdebug отключаем?

**Comment by vmpartner** [comment]
Да, проект старый, я уже и на php то не пишу, чтобы Битриксом заниматься )
Вообще хочется причесать всё, сделать продакшн образы, но время никак не найду свободное.
